plugins/item_tasks/web_client/templates/paginateTasksWidget.pug

```diff
@@ -3,6 +3,6 @@
 
 .list-group.g-task-list-container
   each task in tasks
-    a.list-group-item.g-execute-task-link(href=`#item_task/${task.id}/run`)
+    .list-group-item.g-execute-task-link(data-task-id=task.id)
```

Comment:
In the past, people have stated their preference for using links to navigate, because it makes it easy to open them in a new tab or copy the URL from the browser. One way to achieve both behaviors is to conditionally make it either a link or an event-triggering element based on some variable that is passed into the widget & template.